src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -0,0 +1,2164 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.IO;
+using System.IO.Compression;
+using System.Net;
+using System.Net.Http;
+using System.Net.Security;
+using System.Runtime.InteropServices;
+using System.Security.Authentication;
+using System.Security.Cryptography.X509Certificates;
+using System.Security.Principal;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+using SafeInternetHandle = Interop.WinHttp.SafeInternetHandle;
+
+namespace System.Net.Http
+{
+#if HTTP_DLL
+    internal enum WindowsProxyUsePolicy
+#else
+    public enum WindowsProxyUsePolicy
+#endif
```

Comment:
@bartonjs @stephentoub @mellinoe I did the top (more duplicative) way because of the reason that VS mangles things.  I have no objections to changing it to the bottom way and letting the next developer worry about it when they use 'Format Document' in VS.